ankihub/gui/web/reviewer_buttons.js

```diff
@@ -225,6 +232,65 @@ class AnkiHubReviewerButtons {
         document.head.appendChild(style);
     }
 
+    addResourceCountIndicator(button, buttonName) {
+        const indicator = document.createElement("div");
+        indicator.classList.add("ankihub-reviewer-button-resource-count");
+        indicator.dataset.button = buttonName;
+        this.setResourceCountIndicatorStyles(button, indicator);
+        document.body.appendChild(indicator);
+    }
+
+    setResourceCountIndicatorStyles(button, indicator) {
+        indicator.style.position = "absolute";
+        indicator.style.right = "10px";
+        indicator.style.zIndex = "999";
+        indicator.style.fontSize = "small";
+        indicator.style.borderRadius = "5px";
+        indicator.style.textAlign = "left";
+        indicator.style.fontWeight = "bold";
+        indicator.style.padding = "8px";
+        indicator.style.height = "12px";
+        indicator.style.width = "48px";
+        indicator.style.borderRadius = "92px";
```

Comment:
:wrench: I adjusted the styling according to the values on [Figma](https://www.figma.com/design/rWzQ7PPqAXf1piz5yTntaT/%F0%9F%91%91--Ankihub?node-id=7686-24948&t=ze7e4UhltklLkdt0-1):
```suggestion
        indicator.style.right = "48px";
        indicator.style.zIndex = "999";
        indicator.style.fontFamily = "Merriweather sans-serif";
        indicator.style.fontSize = "12px";
        indicator.style.fontWeight = "800";
        indicator.style.height = "18px";
        indicator.style.width = "16px";
        indicator.style.borderRadius = "20px 0 0 20px";
        indicator.style.display = "flex";
        indicator.style.alignItems = "center";
        indicator.style.justifyContent = "center";

```

Before:
<img src="https://github.com/user-attachments/assets/372f6dfc-2063-4e5f-a1b5-bc018892cace" width="200" />

After:
<img src="https://github.com/user-attachments/assets/a492ff5b-96a6-4068-aa15-d59c3431d19d" width="200" />